marlowe-playground-client/src/Simulation.purs

```diff
@@ -446,61 +458,74 @@ transactionComposer ::
   forall p.
   State ->
   HTML p Action
-transactionComposer state =
-  div [ classes [ ClassName "transaction-composer", ClassName "composer" ] ]
-    [ ul [ class_ (ClassName "participants") ]
-        if (Map.isEmpty possibleActions) then
-          [ text "No valid inputs can be added to the transaction" ]
-        else
-          (actionsForParties possibleActions)
-    , div [ class_ (ClassName "transaction-btns") ]
-        [ ul [ classes [ ClassName "demo-list", aHorizontal ] ]
-            [ li [ classes [ bold, pointer ] ]
-                [ a
-                    [ onClick $ const
-                        $ if hasHistory state then
-                            Just Undo
-                          else
-                            Nothing
-                    , class_ (Classes.disabled $ not isEnabled)
-                    ]
-                    [ text "Undo" ]
-                ]
-            , li [ classes [ bold, pointer ] ]
-                [ a
-                    [ onClick $ const
-                        $ if hasHistory state then
-                            Just ResetSimulator
-                          else
-                            Nothing
-                    , class_ (Classes.disabled $ not isEnabled)
-                    ]
-                    [ text "Reset" ]
-                ]
-            ]
-        ]
-    ]
-  where
-  currentBlock = state ^. (_marloweState <<< _Head <<< _slot)
-
-  isEnabled = isContractValid state
+transactionComposer state
+  | isNothing (state ^. (_marloweState <<< _Head <<< _executionState)) =
+    div [ classes [ ClassName "transaction-composer", ClassName "composer" ] ]
+      [ ul [ class_ (ClassName "participants") ]
+          [ text "Simulation has not started yet" ]
+      , div [ class_ (ClassName "transaction-btns") ]
+          [ ul [ classes [ ClassName "demo-list", aHorizontal ] ]
+              [ li [ classes [ bold, pointer ] ]
+                  [ a
+                      [ onClick $ const $ Just StartSimulation ]
+                      [ text "Start simulation" ]
+                  ]
+              ]
+          ]
+      ]
+  | true =
```

Comment:
standard is to use `otherwise`

Reply:
Ah, I assumed it was different in purescript because it said undefined, I just had to import it. Fixed